src/components/header/Backtrace.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React, { PureComponent } from 'react';
+import { filterCallNodePathByImplementation } from '../../profile-logic/transforms';
+import { getOriginAnnotationForFunc } from '../../profile-logic/profile-data';
+
+import type { Thread, IndexIntoStackTable } from '../../types/profile';
+import type { CallNodePath } from '../../types/profile-derived';
+import type { CauseBacktrace } from '../../types/markers';
+import type { ImplementationFilter } from '../../types/actions';
+
+require('./Backtrace.css');
+
+type Props = {
+  thread: Thread,
+  cause: CauseBacktrace,
+  implementationFilter: ImplementationFilter,
+};
```

Comment:
nit: please define props like this:
```javascript
type Props = {|
  +thread: Thread,
  ...
|};
```
This makes the props stricter (we can't provide more property by mistake) and read-only. Newer Flow (v0.59) has a $ReadOnly utility type for the same thing, but we can use `+` now so let's use it :) I try to use this style in newer components for now but I'd like to convert them all eventually.